### PR TITLE
added 4.15 release notes

### PR DESCRIPTION
**Pull request by tungmhoang**
### Summary

- Preview: http://10.2.7.208/release-notes/all-releases/4_15

### Related PRs, issues, or features (optional)

- https://kobiton.atlassian.net/browse/KOB-46476

### Metadata

- [x] Adds new file(s)
- [x] Edits existing file(s)
- [ ] Removes file(s)

### PR contributor checklist

- [ ] My PR follows the [Kobiton Docs contributor guidelines](https://github.com/kobiton/docs/blob/main/CONTRIBUTE.adoc), meaning:

    - My content contains correct spelling and grammar.
    - My directories and files are [named](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-and-file-names) and [structured](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-structure) correctly.
    - My style and voice follows the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human).
    - I added all new pages to their section's [`nav.adoc` file](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#configure-navigation-in-navadoc).
    - I removed all localizations (like `en-us`) from my URLs.

### CODE REVIEW

File: docs/modules/release-notes/pages/all-releases/4_15.adoc [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+====
+
+* To access the *Other Settings* page, the user must be assigned a role with the permission `org_setting.modify`.
+
+* The *Allow unlimited retention* checkbox applies to the whole org. By default, this checkbox is checked.
+
+====
+
+==  Monitor number of device offline errors in the last hour
```
Comment (erincbailey) [inline docs/modules/release-notes/pages/all-releases/4_15.adoc:46]: Is this working for Cloud now? If not, please comment out.

**Review by erincbailey — CHANGES_REQUESTED** [review]
Fine to publish after reviewing the comment.

File: docs/modules/release-notes/pages/all-releases/4_15.adoc (hunk 1 above)
Comment (tungmhoang) [reply docs/modules/release-notes/pages/all-releases/4_15.adoc:46]: Almost forgot. I will remove it.

**Comment by tungmhoang** [comment]
Feedback addressed. Will merge

**Pull request merged**